src/__tests__/components/ReportPage.test.jsx

```diff
@@ -357,4 +361,19 @@ describe('<ReportPage />', () => {
       expect(component.state('filteredReport')).toEqual([]);
     });
   });
+
+  describe('doSearch method', () => {
+    it(`should call doSearch method if the 
+    searchResults in the state is updated`, () => {
+      const component = getComponent();
+      const doSearch = jest.fn();
+      expect(doSearch).toHaveBeenCalledTimes(0);
+      expect(component.state('searchResult')).toEqual(false);
+      component.setState({
+        filteredReport: [sampleReports[0]],
+        searchResult: true,
+      });
+      expect(component.state('filteredReport')).toEqual([sampleReports[0]]);
+    });
+  });
```

Comment:
The title of this test states: `should call doSearch method if the searchResults in the state is updated`.
However, I can't find any assertion in this test block testing for if `doSearch` method is actually called. Maybe I am missing something, please clarify.

Reply:
@IAMOTZ I have changed the naming of the test because 
I meant to write a test to filter the report if the searchResults is updated in state